scripts/changelog.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Please note that this script only work with Github repositories.
+# Prerequisites: git, github cli
+
+GHCLI_BIN="ghcli"
```

Comment:
On what system is this the default name for the GitHub CLI? Is this https://github.com/cli/cli, commonly known on PATH as `gh`? I had to edit the script to test this because GHCLI_BIN is not overridable, but before we make it overridable, is there any reason we can't hardcode `gh` instead?

Reply:
No `ghcli` was just an alias from my env. I'm wondering if we should add a few lines to install the GH cli if it's not already installed (targeting a fixed version), and put the binary in `bin`.

Reply:
Personally, I absolutely hate it when development scripts install tools for me 😅. I think we're overthinking this: if the github CLI breaks its interface, let's have a think about version pinning then. In the mean time, how about we just call `gh`?

Reply:
> I absolutely hate it when development scripts install tools for me

I agree, we should absolutely avoid to do that. 
What about a docker container if we want to provide a consistent environment, maybe this is overkill for that ? 
Can we try not to use gh CLI at all ?

Reply:
Requiring docker would certainly be overkill. IMO, making scripts sufficiently parameterisable to be runnable with `docker run` is fine, but we shouldn't need linux containers to run a changelog builder.